scripts/rollup/build.js

```diff
@@ -506,9 +407,8 @@ function getPlugins(
           gzip,
         };
       },
-    })
-  );
-  return plugins;
+    }),
+  ].filter(Boolean);
```

Comment:
I’m not sure this is super obvious as to what’s happening?

Reply:
I dunno, seems like a common pattern with plugin systems?

<img width="423" alt="screen shot 2017-11-08 at 08 11 31" src="https://user-images.githubusercontent.com/810438/32538010-729a4382-c45c-11e7-913c-f3f497f76138.png">

Happy to change to something explicit if you prefer.